### PR TITLE
Issue 5 Image Tags PR

### PR DESCRIPTION
**Pull request by nair-amrita**
Closes Issue#5

- When I add a new image, there is a form field to add tags.
- After I've saved an image with tags, I see those tags displayed with the associated image on the show page
- My application uses the acts-as-taggable-on gem to provide the tagging functionality.
- Next to each image on the index page, the tags for that image should be displayed

### CODE REVIEW

**Comment by nair-amrita** [comment]
[Issue 5: Image Tags](https://trello.com/c/dxsYCSLS/333-issue-5-image-tags)

File: app/views/image_posts/new.html.erb [hunk 1]
```diff
@@ -9,12 +9,15 @@
...
   <p>
-    <%= form.label :title %><br>
+    <%= form.label :title %>
     <%= form.text_field :title %>
   </p>
-
   <p>
-    <%= form.label :image %><br>
+    <%= form.label :tags %>
+    <%= form.text_field :tag_list %>
```
Comment (steventhan) [inline app/views/image_posts/new.html.erb:17]: Does this allow multiple tags per form submit?

File: app/views/image_posts/index.html.erb [hunk 2]
```diff
@@ -6,6 +6,8 @@
     <% @posts.order("created_at DESC").each do |post| %>
         <%= post.title %>
         <br/>
+        <h5 id="tags">Tags: <%= post.tag_list %></h5>
```
Comment (mertdusunceli) [inline app/views/image_posts/index.html.erb:9]: nitpicking but let's not show 'Tags:' if the tag_list is empty

File: app/models/image_post.rb [hunk 3]
```diff
@@ -1,4 +1,5 @@
 class ImagePost < ApplicationRecord
+  acts_as_taggable_on :tags
```
Comment (steventhan) [inline app/models/image_post.rb:2]: Since we make changes to model, it should be tested to verify the behavior

File: config/routes.rb [hunk 4]
```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   get 'welcome/index'
-
+  get '/tagged', to: 'image_posts#tagged', as: :tagged
```
Comment (steventhan) [inline config/routes.rb:3]: This new route should be tested too!

File: app/views/image_posts/new.html.erb (hunk 1 above)
Comment (nair-amrita) [reply app/views/image_posts/new.html.erb:17]: yes by comma separating the tags it allows multiple tags.

File: config/routes.rb (hunk 4 above)
Comment (nair-amrita) [reply config/routes.rb]: This was to be removed.

File: app/views/image_posts/index.html.erb (hunk 2 above)
Comment (nair-amrita) [reply app/views/image_posts/index.html.erb]: makes a lot more sense. thanks!

**Pull request merged**